src/store/workflows.module.js

```diff
@@ -276,10 +265,11 @@ function cleanParents (state, node) {
  */
 function applyInheritance (state, node) {
   if (node.type === 'family' && node.node.childTasks) {
-    // add new tasks
+    // build a mapping of {childID: childNode} for faster lookup
+    const childIDs = node.children.reduce((map, obj) => { map[obj.id] = obj; return map }, {})
```

Comment:
Similary to Python dictionaries, object lookups are `O(1)` whereas iterating over the array is `O(N)`.

Reply:
Bravo! :clap: